### ISSUE TITLE
Trapper requires maps designed for it to be of any use

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/gameplay#16]
Trapper, which is very important for aliens which do not want to give easy momentum (by building barricades to block all accesses, this cost too much and can feed humans from afar), require maps with specific spots to put them.

Those spots needs to be close enough to ground so that the trapper can hit the target:

* speed of 500
* not affected by gravity AFAIK
* linear trajectory

It's also requires the spot to be big enough so that humans won't be able to fire at it too easily, which means the spot must be bigger than a 30x30x30 square if I understand correctly content of trapper.model.cfg.
A spot of that exact size imply humans will be able to see or shoot at it easily, especially if not covered by a door.
Also, trappers have a creepSize value of 48, increasing chances that they will be detected, when this structure is really fragile (50 health) but still have a non-negligible cost (8BP)?

For what it's worth, a human _walking forward_ have a measured speed of 319, unaffected by armor. Sprinting or jumping will increase that speed, whatever the armor. When sprinting, it has been measured to be 383 for naked and light armors, or 367 for medium and heavy armors.
Numbers common for all humans that can be found in configuration are:

* speed = 1.0
* acceleration = 10.0
* friction = 7

What results of current trapper's configuration is that it's so easy to misplace it that unwary humans (including bots) will regularly "dodge" the lockblob, hear or see the blob hitting ground, thus guessing there's something, move a bit because trapper's area is not that wide, search and destroy the trapper.

**Comment by bmorel** [comment]
FWIW I gave a quick try at increasing blob's speed, to no good results. Too slow and it never hits, too fast and it never hit (but hit forward the human!).

**Comment by Viech** [comment]
Maybe when it was too fast, you haven't updated some pre-aiming/movement tracking code?

**Comment by bmorel** [comment]
Yep, I really did a *quick* try by changing the config value. I didn't tried very hard.

**Comment by bmorel** [comment]
I'm thinking, another quick'n dirty experiment would be to drastically increase the missile's size. Bigger missiles have obviously easier time to hit, even if this means they might also hit something unexpected in the way.
